samples/nrf9160/modem_shell/src/main.c

```diff
@@ -155,6 +166,11 @@ void main(void)
 	modem_info_params_init(&modem_param);
 #endif
 
+	err = dk_buttons_init(button_handler);
+	if (err) {
+		printk("Failed to initialize DK library, error: %d", err);
```

Comment:
To be more specific, I would say `DK buttons library` as we are using `dk_buttons_init()`